locales/en/breaches.ftl

```diff
@@ -15,6 +15,8 @@ emails-monitored =
      *[other] { $count } of { $total } emails monitored
   }
 
+cta-add-email = Add email address
```

Comment:
Same as for the other PR, let's use more semantic IDs. `cta` is jargon, `add-email-link` would be clearer.